app/views/content/life-as-a-teacher/design-and-technology/_article.html.erb

```diff
@@ -0,0 +1,76 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach design and technology</h2>
+    <p>Design and technology is a broad subject. Your teaching will cover a diverse range of areas including maths, science, engineering, computing and art. You'll encourage pupils to solve problems creatively and consider how the technology they use can impact their designs and the environment.</p>
+  </section>
+
+  <section class="col col-720">
+    <%= render Content::QuoteComponent.new(
+    text: "People may not realise how much potential there is for progression in teaching. They think they go into the world of the classroom and that's it. Lots of teachers have added responsibilities they enjoy doing and find fulfilling.",
+    name: "Chris Farr, teacher training adviser",
+    large: true
+  ) %>
+  </section>
+
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Shape your pupils' futures</h2>
+      <p>Teaching design and technology opens the door for your pupils to pursue exciting careers in fields as diverse as interior design and architecture to engineering and joinery.</p>
+      <p>By solving challenging problems, your pupils could make significant contributions to these diverse and impactful areas.</p>
+      <p>Your pupils will learn more from you that you might think.  As their teacher, you will guide your pupils though the skills they need to show their understanding of design and technology. You will teach them valuable life skills such as critical thinking, decision making and communication.</p>
+  </section>
+
```

Comment:
```suggestion
    <section class="col col-720">
    <%= render Content::QuoteComponent.new(
    text: "My passion is simple: food. It brings everyone together; we can all connect over it. So making the decision to teach food tech was simple too. Since I began teaching, I’ve focused on how my lessons can expand the horizons of my students – using cooking to build their confidence.",
    name: "Andrew, food technology teacher",
    large: true
  ) %>
  </section>
```